src/python/Dockerfile

```diff
@@ -56,17 +56,6 @@ RUN set -xe \
   && pip install --target=/contrast "contrast-agent==${VERSION}" \
   && echo "{ \"version\": \"${VERSION}\" }" > /contrast/image-manifest.json
 
-FROM python:3.8-slim-bookworm AS builder-38
```

Comment:
I still hate codenames, even when third parties use them. `bullseye` is now `bookworm`, :coolcoolcool: ?